src/upgrade.rs

```diff
@@ -30,6 +30,8 @@ enum DepName {
     CairoLS,
     #[value(name = "cairolint")]
     CairoLint,
+    #[value(name = "stwocairo")]
+    StwoCairo,
```

Comment:
```suggestion
    #[value(name = "stwo")]
    Stwo,
```

don't we want this to be just like this?

Reply:
discussed offline